gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Thu Mar 29 15:52:32 IST 2018
+#Sun Feb 24 00:16:32 IST 2019
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.6-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.1-all.zip
```

Comment:
Revert

Reply:
I didn't change this, maybe my android is not update so this happened.

Reply:
@sushmita0810 You made mistakes while importing the project. You shouldn't have update the distributed gradle version. Reverting to a previous commit and then redoing the PR changes may help.